docs/run-basic.md

```diff
@@ -0,0 +1,93 @@
+# Running the OMS Locally
+
+Follow these instructions to run the OMS locally, using the 
+Temporal Service provided that the `temporal` command. This 
```

Comment:
typo

Reply:
Fixed in https://github.com/temporalio/reference-app-orders-go/commit/f57c5416ebdc442a80d34830ce52b27e4f19714b